#!/usr/bin/env python
PACKAGE = "yolo_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Thresh",    double_t,    0, "People: Main Yolo Confidence Threshold", 0.25,  0, 1)
gen.add("Hier_Thresh", double_t, 0, "People: Additional Yolo Confidence Threshold",    0.5, 0,   1)
gen.add("Median_Factor",    double_t,    0, "People: How large to scale box around center scaled against detection size",  0.2, 0, 1)


gen.add("ObjectThresh",    double_t,    0, "Objects: Main Yolo Confidence Threshold", 0.4,  0, 1)
gen.add("ObjectHier_Thresh", double_t, 0, "Objects: Additional Yolo Confidence Threshold",    0.5, 0,   1)
gen.add("ObjectMedian_Factor",    double_t,    0, "Objects: How large to scale box around center scaled against detection size",  0.1, 0, 1)


exit(gen.generate(PACKAGE, "yolo_detector", "open_ptrack_yolo"))
